website/static/js/fangorn.js

```diff
@@ -607,9 +607,12 @@ function _fangornLazyLoadOnLoad (tree) {
  * @private
  */
 function _fangornOrderFolder(tree) {
-    var sortDirection = this.isSorted[0].desc ? 'desc' : 'asc';
-    tree.sortChildren(this, sortDirection, 'text', 0);
-    this.redraw();
+    // Checking if this column does in fact have sorting
+    if (this.isSorted[0]) {
```

Comment:
I found it unintuitive that `isSorted` was *not* a Boolean. I think this is a minor naming issue: perhaps `tb.sortOptions` would be more appropriate.

Not a big deal; this can be deferred.